backend/lib/endpoints/posts.js

```diff
@@ -276,12 +276,12 @@ async function routes(app) {
       schema: deletePostSchema,
     },
     async (req) => {
-      const { userId } = req.body;
+      const { userId } = req;
       const { postId } = req.params;
       const [findErr, post] = await app.to(Post.findById(postId));
       if (findErr) {
         throw app.httpErrors.notFound();
```

Comment:
maybe out of scope here but wonder if the `findErr` should be logged instead of just returning  "not found" (I think I've mentioned this before on another PR, this occurs multiple times in similar checks)

Reply:
Also out of scope but this logic (not just here but across the app) seems a bit incomplete.

```
      const [findErr, post] = await app.to(Post.findById(postId));
      if (findErr) {
        throw app.httpErrors.notFound();
```
If I pass an invalid ObjectId there is an error - like posts/1

BUT if I pass a valid ObjectId that doesn't exist both are null so I get, for example:

![image](https://user-images.githubusercontent.com/10546720/82828611-763d8580-9e7f-11ea-8be3-15a092b8e66d.png)

should it be `if (findErr || post === null)` ?

Reply:
Good point. 
What you say makes sense @joshmorel
Did you open an issue for that?